### ISSUE TITLE
It appears that you have a fresh installation. How would you like to initialize this node?

### ISSUE THREAD

**Issue by meierw** [AxonIQ/axon-server-se#654]
Prior to Axon Server 2023.x, whenever I would run Axon server in a container, from the `axoniq/axonserver` image, it would just work straight away.

Now it's showing this message on first run inside the web console. And doesn't allow connecting to the server right away.

```
It appears that you have a fresh installation. How would you like to initialize this node?
Would you like to complete the installation and use this node as a standalone node or as the first node in a cluster (Enterprise Edition only), or as a node in an existing cluster?
Action:

Start as standalone node or first node
With context name:
default

Finish
```

This is an issue, as we relied on the server working right away after boot in many of our dev environments and testcontainers.

What would be a quick solution to the issue? Do I need to include the `axonserver.properties` file in my container with some extra defaults configured? I couldn't figure from the docs, if this is even possible with config properties.

**Comment by bert-laverman** [comment]
Hey Walter!
Can you please add the exact tags you have been using? There are two different "styles" available from the repository, with one running in the root ("`/`") and the other running in "`/axonserver`". Some images also run as user "`axonserver`", so it is important to be careful about this. The images marked with "`-dev`" are of the "`/axonserver`" variety, while the ones without are of the "`/`" variety.

As to why this difference exists: the "dev" images are built using the Eclipse Temurin images and support both Intel/AMD as well as ARM processor architectures, while the others use the older "Distroless" images that do not. Given the popularity of ARM silicon nowadays, we need to support that.

If this is indeed the problem you are facing, you can either remap your volumes to the new locations or build your own image.

**Comment by MGathier** [comment]
There are properties you can set to make Axon Server initialize itself immediately on first start. These are:
axoniq.axonserver.autocluster.first=<your-docker-hostname>
axoniq.axonserver.autocluster.contexts=default

You can add these to the axonserver.properties or add them as environment properties.

**Comment by meierw** [comment]
Hi @bert-laverman I was using `-dev` both before and after the version bump to 2023.x

@MGathier That indeed looks like what I was searching for. Can you share where in the docs you found those? I think I must be blind for missing that.

**Comment by bert-laverman** [comment]
Thank you for this report. I can confirm the behavior. I suggest you set the container's hostname explicitly and use that as the value for "`AXONIQ_AXONSERVER_AUTOCLUSTER_FIRST`." So, add "`hostname: axonserver`" to your docker-compose file and (assuming you set no domain) use "`AXONIQ_AXONSERVER_AUTOCLUSTER_FIRST=axonserver`", comparable to:

```
docker run -d --name axonserver --hostname axonserver -p 8024:8024 -p 8124:8124 -e AXONIQ_AXONSERVER_AUTOCLUSTER_FIRST=axonserver -e AXONIQ_AXONSERVER_AUTOCLUSTER_CONTEXTS=default axoniq/axonserver:2023.1.0-dev
```

At startup, Axon Server compares this "first" value to its own configured FQDN and will start cluster initialization only if they match. If they do not match, it will attempt to contact the provided host and join its cluster instead, which is a licensed functionality.

**Comment by schananas** [comment]
We have released Axon Server 2023.1.1 with a change to simplify this and resolve issue:

2023.1.1 Update:
To simplify initialization of Axon Server, it now supports a new property “axoniq.axonserver.standalone=true”. When this property is set on a clean Axon Server instance it initializes the server with a “default” context.

No need to set auto cluster properties anymore, in fact, new property should not coexist with auto cluster or cluster template configuration

**Comment by meierw** [comment]
@schananas 

Nice!. Big thanks, this will make our docker-compose and testcontainer logic a lot neater.
